t/Legacy/More.t

```diff
@@ -196,6 +196,10 @@ is_deeply( { foo => $sub, bar => [1, $glob] },
 is_deeply( qr/a/, qr/a/, "same regex" );
 
 
-# These two tests must remain at the end.
-is( $@, $Err,               '$@ untouched' );
-cmp_ok( $!, '==', $Errno,   '$! untouched' );
+SKIP: {
```

Comment:
I think it would be better to fix this issue. It should just require localizing these variables around the call to `warnings::enabled` that was recently added.

Reply:
Done.